### DISCUSSION TITLE
下载过程中改变需要下载的文件，是否会触发进度检查器

### DISCUSSION THREAD

**Discussion by asterho** [PBH-BTN/PeerBanHelper#621] [问答] [answered]
有时不需要下载种子里的所有文件，只需要一部分，这种改变所需文件的进而改变下载进度的操作，是否会触发PBH的进度检查器？

**Comment by 135ty** [comment]
参见#590

**Reply by asterho** [reply]
看文档的描述，进度检查器似乎会同时记录peer的（汇报的）下载进度和（通过记录上传量计算出的）下载进度，这个计算是否考虑了中途改变选取文件的情况？我认为 #590 并没有解答这一点。

**Reply by 135ty** [reply, marked as answer]
你改变下载的文件会对进度有影响，只要进度是倒退的检测到就会被ban，不会考虑到改文件的情况。
